vertx-core/src/main/java/io/vertx/core/impl/deployment/DefaultDeploymentManager.java

```diff
@@ -27,6 +28,8 @@ public class DefaultDeploymentManager implements DeploymentManager {
 
   public static final Logger log = LoggerFactory.getLogger(DefaultDeploymentManager.class);
 
+  private static final AtomicLong nextId = new AtomicLong();
```

Comment:
the idea of cache dir is to avoid that, and keep the same behaviour we have, so please no.

Reply:
So what do you propose?

Reply:
I believe you confused two changes @vietj  : the cache dir that used a random UUID, and the verticle id generator

Reply:
sorry, can we have the verticle id generator in another PR then ?

Reply:
I'd like to keep distinct PR for the changelog

Reply:
PR

Reply:
https://github.com/eclipse-vertx/vert.x/pull/5469

Reply:
PR updated